src/defaults.js

```diff
@@ -2,9 +2,13 @@ import parsers from './parse/parsers';
 
 const defaults = {
   beautifier: {
-    indent_size: 1,
-    indent_char: '	',
-    indent_with_tabs: true
+    indent_size: 2,
+    indent_char: ' ',
+    indent_with_tabs: false,
+    unformatted:
```

Comment:
I'm on the fence about how I feel about this trick :). I get the space (and quote-typing) savings, but wonder if it's a little counter-intuitive. Eh. I don't feel passionately!